### PR TITLE
NXBT-1305: shibboleth service provider docker image

### PR DESCRIPTION
**Pull request by funshodavid**

### CODE REVIEW

**Comment by jcarsique** [comment]
What is the source of the copy/paste? I though that it was https://github.com/nuxeo/nuxeo-tools-docker/tree/master/openldap but the first lines of the Jenkinsfile are not consistent (cf timeout vs node)

File: shibboleth-sp/Jenkinsfile
Comment (jcarsique) [inline shibboleth-sp/Jenkinsfile]: wrong order (timeout vs node)

File: shibboleth-sp/Jenkinsfile
Comment (jcarsique) [inline shibboleth-sp/Jenkinsfile]: wrong

**Review by jcarsique — CHANGES_REQUESTED** [review]
Please rebase compared to the current status of the source example, all previous comments to https://github.com/nuxeo/nuxeo-tools-docker/tree/master/openldap seem applicable here again

Regarding the TODO, you cannot spread the debt: it was up to @ffjdm to fix and remove the TODO or create the related JIRA tickets and reference them in the TODO. As is, you cannot copy the TODO.

**Comment by jcarsique** [comment]
:+1: 
Can you plan with @ffjdm to replace the TODO with TODO NXBT-9999 ? Before or after merge, but soon so that the TODO don't stay as is forever.

**Review by jcarsique — APPROVED** [review]

**Review by ffjdm — APPROVED** [review]
👍

**Pull request merged**
